examples/firestore/firestore_test.go

```diff
@@ -0,0 +1,63 @@
+package firestore
+
+import (
+	"cloud.google.com/go/firestore"
+	"context"
+	"os"
+	"testing"
+)
+
+type Person struct {
+	Firstname string `json:"firstname"`
+	Lastname  string `json:"lastname"`
+}
+
+func TestFirestore(t *testing.T) {
+	ctx := context.Background()
+
+	container, err := setupFirestore(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Clean up the container after the test is complete
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	os.Setenv("FIRESTORE_EMULATOR_HOST", container.URI)
```

Comment:
The testing library supports setting the environment only for that test
```suggestion
	t.Setenv("FIRESTORE_EMULATOR_HOST", container.URI)
```